src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -5214,20 +5214,20 @@ public void AddTask(string name, int value)
 
         public void AddKeyword(string name, ulong value)
         {
-            if ((value & (value - 1)) != 0)   // Is it a power of 2?
+            if ((value & (value - 1)) != 0) // Must be zero or a power of 2
```

Comment:
I wonder if its worth having a `IsPow2OrZero` function to cover this other "common" pattern.

It might likewise be interesting to see if the JIT does the "right thing" for `IsPow2(value) || (value == 0)`

Reply:
As I was making these changes, I went looking to see if we had that method :-) I think it'd be worth adding, not just because it's useful sporadically, but because it highlights that IsPow2 doesn't return true for zero.